pkgs/applications/misc/mu-repo/default.nix

```diff
@@ -1,25 +1,30 @@
-{ lib, fetchFromGitHub, buildPythonApplication, pytestCheckHook, git }:
+{ lib, fetchFromGitHub, buildPythonApplication, pytestCheckHook, git, testers, mu-repo }:
 
 buildPythonApplication rec {
   pname = "mu-repo";
-  version = "1.8.1";
+  version = "1.8.2";
 
   src = fetchFromGitHub {
     owner = "fabioz";
-    repo = pname;
+    repo = "mu-repo";
     rev = "mu_repo_${lib.replaceStrings [ "." ] [ "_" ] version}";
-    sha256 = "0mmjdkvmdlsndi2q56ybxyz2988ppxsbbr1g54nzzkkvab2bc2na";
+    hash = "sha256-COc7hbu72eA+ikZQkz6zXtFyaa/AKhoF+Zvsr6ZVOuY=";
   };
 
   propagatedBuildInputs = [ git ];
 
   checkInputs = [ pytestCheckHook git ];
 
+  passthru.tests.version = testers.testVersion {
+    package = mu-repo;
+  };
+
   meta = with lib; {
     description = "Tool to help in dealing with multiple git repositories";
     homepage = "http://fabioz.github.io/mu-repo/";
     license = licenses.gpl3;
```

Comment:
Fix license - gpl3Plus or gpl3Only.

Reply:
I don't know exact one, there is no details about it in the upstream repo.